docs/reference/rule_attributes.md

```diff
@@ -0,0 +1,68 @@
+# Ick Toml Reference
+The following attributes can be set in an `ick.toml` or `pyproject.toml`.
+
+## `[[ruleset]]` Attributes
+
+### Required (at least one)
+- `url` (str): A URL to an external repository containing rules
+- `path` (str): A local path to a directory containing rules
+
+### Optional Attributes
+- `prefix` (str): A prefix to use for the rules from this mount. If not specified, it will be derived from the last component of the URL or path.
+
+
+## `[[rule]]` Attributes
+
+A `RuleConfig` (defined in `ick/config/rules.py`) defines the configuration for a single `[[rule]]` in the system. Here are all available attributes:
+
+### Required Attributes
+
+- `name` (str): The name of the rule
+- `impl` (str): The language in which the rule will be written, see [impls] (TODO)
+
+### Optional Attributes
+
+#### Execution Control
+- `scope` (Scope): The scope of the rule's execution. Defaults to single file. Available options:
```

Comment:
Here the type is shown as `Scope`, but that's an internal class name.  The user has to provide a string.